nfsn-setup.sh

```diff
@@ -19,7 +19,25 @@ nfsn -s list-aliases > letsencrypt.sh/domains.txt
 echo " + Performing initial run..."
 letsencrypt.sh/letsencrypt.sh --cron
 
-echo " + Done! Now add $(realpath nfsn-cron.sh) to your scheduled tasks!"
-echo "   You can find them at https://members.nearlyfreespeech.net/$(echo ${MAIL} | cut -d '/' -f4 | cut -d'_' -f1)/sites/${NFSN_SITE_NAME}/cron"
-echo "   letsencrypt.sh will not try and renew the script unless it expires in less than 30 days."
-echo "   I recommend running it daily, to ensure plenty of warning if the renewal fails."
+user_site=${MAIL##*/}
+printf '
+   Now add nfsn-cron.sh to your scheduled tasks so that the certificates will be
+   renewed automatically.  To do that, go to
+
+       https://members.nearlyfreespeech.net/%s/sites/%s/cron
+
+   and use the following settings:
+
+       Tag:                  lets-nfsn
+       URL or Shell Command: %q %q
+       User:                 me
+       Hour:                 %d
+       Day of Week:          Every
+       Date:                 *
+
+   The certificates will be renewed only when needed so it’s safe to schedule
+   the task to run daily.
+' \
+       "${user_site%_*}" "$NFSN_SITE_NAME" \
+       "$BASH" "$(realpath nfsn-cron.sh)" \
+       "$(( $RANDOM % 24 ))"
```

Comment:
Now this, this I like, though I'll need to add a note about cron being broken until `/usr/local/bin/nfsn` is updated.